client/src/components/App.jsx

```diff
@@ -223,47 +221,54 @@ class App extends React.Component {
             showModal: false,
             dimClass: `none`
           })
-        }, Number(animation.dimDuration));
-    })});
+        }, Number(Animation.dimDuration))
+      });
+    });
   }
 
   render() {
-
     // pop-up review modal
-    const ReviewModal = this.state.showModal ? (<StyledAppModal ref={this.modal} reviews={this.state.reviews} ratings={this.state.ratings} close={this.closeModal.bind(this)} />) : null;
+    const ReviewModal = this.state.showModal
```

Comment:
I'm curious as to what this line is doing. Im sure there is a reason for it.
Usually mine would like something like:
const  { showModal } = this.state;

Reply:
good point, I will change it to what you suggested. The line is supposed to hold the modal component if the state says it's supposed to